### DISCUSSION TITLE
cannot import name 'Compose' from 'mmaction.datasets.transforms'

### DISCUSSION THREAD

**Discussion by pipixin321** [open-mmlab/mmaction2#2518] [Q&A]
I try to use tools/data/activitynet/tsn_feature_extraction.py  to extract feature from video frames. However, 'Compose' doesn't exsit in mmaction.datasets.transforms. Besides, the same import erro happens in ' from mmaction.models import build_model' . I wonder if any solution exisit to successfully import these two methods.

**Comment by cir7** [comment]
Which version are you using? the script is for 0.x version, which is incompatible with the main branch.
